js/bybit.js

```diff
@@ -0,0 +1,1001 @@
+        // so why not ask user to point out in  which order to cancel ?
+        let response = undefined;
+        if ('stop_order_id' in params) {
+            response = await this.privatePostStopOrderCancel (params);
+        } else if ('order_id' in params) {
+            response = await this.privatePostOrderCancel (params);
+        } else {
+            throw new ArgumentsRequired (this.id + " cancelOrder() requires additional argement either 'order_id' to cancel active order or 'stop_order_id' to cancel conditional order. Values of these arguments should be equal to value of 'id' argument.");
+        }
+        return this.parseOrder (this.safeValue (response, 'result', {}));
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'wallet_fund_type': 'Deposit',
+        };
+        if (since !== undefined) {
+            request['start_date'] = this.ymd (since);
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+            request['coin'] = currency['code'];
+        }
+        const response = await this.privateGetWalletFundRecords (this.extend (request, params));
+        const result = this.safeValue (response, 'result', {});
+        return this.parseTransactions (this.safeValue (result, 'data', []), currency, since, limit);
+    }
+
+    async fetchWithdrawals (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+            request['coin'] = currency['code'];
+        }
+        if (since !== undefined) {
+            request['start_date'] = this.ymd (since);
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privateGetWalletWithdrawList (this.extend (request, params));
+        const result = this.safeValue (response, 'result', {});
+        return this.parseTransactions (this.safeValue (result, 'data', []), currency, since, limit);
+    }
+
+    async fetchTransactions (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (since !== undefined) {
+            request['start_date'] = this.ymd (since);
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+            request['coin'] = currency['code'];
+        }
+        const response = await this.privateGetWalletFundRecords (this.extend (request, params));
```

Comment:
This endpoint does not return info for `fetchTransactions`, instead it returns the info for `fetchLedger`:

- https://github.com/ccxt/ccxt/wiki/Manual#ledger